.vscode/settings.json

```diff
@@ -3,6 +3,12 @@
     "editor.fontLigatures": true,
     "editor.snippetSuggestions": "top",
     "editor.rulers": [80, 120],
+    // Set the default
```

Comment:
```suggestion
     "_comment": "Set the default",
```

Reply:
This is not needed here. But it is a good idea for default json files. Thx.